xbmc/LangInfo.cpp

```diff
@@ -275,6 +275,14 @@ void CLangInfo::CRegion::SetGlobalLocale()
 #endif
   }
 
+  try
+  {
+    g_langInfo.m_originalLocale = std::locale(strLocale.c_str());
+  }
+  catch (...) {
```

Comment:
According to docs it throws std::runtime_error "if the operating system has no locale named std_name". Might be worth logging it, not just silence it, so it's easier to track down..